### PR TITLE
Python package cleanup

### PR DESCRIPTION
**Pull request by chris-allan**
Moves the built-in filters out of the root module namespace and fixes import of non-root modules. This PR is backwards compatible with all Parade installations unless the user has added `annotation_filters` or `table_filters` to their `omero.web.parade.filters` setting themselves.

Once this is merged we should be in a good place to mint a `0.1.0` release.

/cc @emilroz

### CODE REVIEW

**Comment by will-moore** [comment]
Working for me and code cleanup makes sense.
This sentence in the README needs a minor tweak "same directory":
"The current default setting lists the omero_parade app itself and two other modules that are in the same directory and are therefore expected to be on the PYTHONPATH when the app is installed.

**Comment by will-moore** [comment]
Playing some more locally, I've got annotation_filters not importing correctly in views.py ```def filter_list```. Probably got some state mixed up in switching branches etc. - Will try to figure it out later...

But, it reminds me that we could add a ```try/except ImportError``` in ```def filter_list()``` the same as is in ```dataprovider_list()``` since then it would be possible to add a module to ```parade_settings.PARADE_FILTERS``` that was only a data provider, not a filter. Currently you MUST have a filter but not a data_provider.

**Comment by chris-allan** [comment]
Is 0b667915a4ad4870b7b1535fac78310855bc6c69 more or less what you had in mind for the `try` block @will-moore?

File: omero_parade/views.py [hunk 1]
```diff
@@ -161,9 +159,9 @@ def get_data(request, data_name, conn=None, **kwargs):
...
     for m in modules:
         try:
-            module = __import__('%s.data_providers' % m)
-            if hasattr(module.data_providers, 'get_dataproviders'):
-                dp = module.data_providers.get_dataproviders(request, conn)
+            module = __import__('%s.data_providers' % m, fromlist=[''])
+            if hasattr(module, 'get_dataproviders'):
+                dp = module.get_dataproviders(request, conn)
                 if data_name in dp:
                     data = module.data_providers.get_data(request, data_name,
```
[the remark this answers is not in the document]
Comment (chris-allan) [reply omero_parade/views.py]: Depends what you want. Right now it's unchanged from the original implementation as far as I read it. If you configure `foo` in your settings it is expecting a `foo.data_providers` module to exist and for it to contain a `get_providers` function.

Before a `foo.bar` module could not be used and this PR fixes that but I don't think that has any impact on the intent.

Happy for this all to work any way really.

[the remark this answers is not in the document]
Comment (chris-allan) [reply omero_parade/views.py]: Gotcha, was going too fast reading your initial comment. I think the latest change reflects exactly that.

**Pull request merged**